.github/workflows/shared.yml

```diff
@@ -26,6 +26,7 @@ jobs:
   test:
     runs-on: ${{ matrix.os }}
     timeout-minutes: 10
+    continue-on-error: true
```

Comment:
I put it up because it makes more sense since it's a job configuration, not a step one.